include/reactor-uc/schedulers/dynamic/scheduler.h

```diff
@@ -42,6 +42,12 @@ struct DynamicScheduler {
    * tag.
    */
   void (*run_timestep)(Scheduler *self);
+
```

Comment:
But isn't this precisely what we tried to avoid? That the stuff that we need for federated creeps into all the corners? (in this case the scheduler)

Reply:
Good point!